src/pages/Game.tsx

```diff
@@ -1,20 +1,22 @@
-import React, {useState, useEffect} from 'react'
-import { DeckResponse } from '../global'
-import { newDeckShuffledLink, decksCount } from '../const/api';
-import GameInterface from '../partials/GameInterface';
-import axios, { AxiosResponse } from 'axios';
+import React, { useContext, useEffect } from 'react';
+
+import GameInterface from '../partials/game/GameInterface';
+
+import { cardService } from '../services/CardService';
+import { GameContext } from '../context/GameContext';
 
 export default function Game() {
 
-  const [deckId, setDeckId] = useState('')
+  const gameContext = useContext(GameContext);
```

Comment:
możesz to wrzucić do pliku z context na zasadzie:
`export useGameContext = () => useContext(GameContext)`
i używać już tylko 
`const gameContext = useGameContext();`
Zawsze to 1 import mniej